src/shared/sources/pl/index.js

```diff
@@ -0,0 +1,84 @@
+const parse = require('../_lib/parse.js')
+const maintainers = require('../_lib/maintainers.js')
+
+const isoMap = require('./mapping.json')
+
+const country = "iso1:PL"
+
+module.exports = {
+    country,
+    maintainers: [ maintainers.qgolsteyn, maintainers.ciscorucinski ],
+    friendly: {
+        name: 'Ministry of Health of the Republic of Poland',
+        url: 'http://www.mz.gov.pl/'
+    },
+    scrapers: [
+        // TODO (scraper) PL: 2020-03-18 scraper
+        // {
+        //     startDate: '2020-03-18',
+        //     crawl: [
+        //         {
+        //             type: 'csv',
+        //             url: 'https://raw.githubusercontent.com/covid19-eu-zh/covid19-eu-data/master/dataset/covid-19-pl.csv'
+        //         }
+        //     ],
+        //     scrape (data, date) {
+        //         const casesByRegion = {}
+        //         const deathsByRegion = {}
+        //
+        //         for (const item of data) {
+        //             const {datetime, nuts_2} = item;
```

Comment:
destructured to remove lint warnings below.

Reply:
You can also `// eslint-disable-next-line` (or something like that) -- it's sometimes necessary.